datafusion/common/src/scalar.rs

```diff
@@ -5912,285 +5859,6 @@ mod tests {
         check_array(array);
     }
 
-    fn get_timestamp_test_data(
```

Comment:
This test is removed as timestamp arithmetic no longer returns intervals (which avoids this whole mess of edge cases)